--- /home/arturas/Builds/dwm/config.def.h	2016-06-19 23:21:17.462397099 +0300
+++ config.h	2016-01-25 19:24:35.494240265 +0200
@@ -10,8 +10,6 @@ static const char selbgcolor[]      = "#
 static const char selfgcolor[]      = "#efefef";
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
-static const unsigned int systrayspacing = 2;   /* systray spacing */
-static const Bool showsystray       = True;     /* False means no systray */
 static const Bool showbar           = True;     /* False means no bar */
 static const Bool topbar            = True;     /* False means bottom bar */
 
@@ -19,12 +17,9 @@ static const Bool topbar            = Tr
 static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
 
 static const Rule rules[] = {
-	/* class                  instance    title       tags mask     isfloating   monitor */
-	{ "Firefox",              NULL,       NULL,       1 << 0,       False,       -1 },
- 	{ "st",                   NULL,       NULL,       1 << 1,       False,       -1 },
- 	{ "chromium-browser",     NULL,       NULL,       1 << 2,       False,       -1 },
- 	{ "MPlayer",              NULL,       NULL,       1 << 8,       False,       -1 },
- 	{ "mpv",                  NULL,       NULL,       1 << 8,       False,       -1 },
+	/* class      instance    title       tags mask     isfloating   monitor */
+	{ "Firefox",  NULL,       NULL,       1 << 0,       False,       -1 },
+ 	{ "st",       NULL,       NULL,       1 << 1,       False,       -1 },
 };
 
 /* layout(s) */
